patches/app-builder-lib+22.7.0.patch

```diff
@@ -0,0 +1,251 @@
+diff --git a/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js b/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
+index 1ff7e89..9442d28 100644
+--- a/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
++++ b/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js
+@@ -135,7 +135,7 @@ class LinuxTargetHelper {
+         exec += executableArgs.join(" ");
+       }
+ 
+-      exec += " %U";
++      exec += " --no-sandbox %U";
+     }
+ 
+     const desktopMeta = {
+diff --git a/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js.orig b/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js.orig
```

Comment:
did you generate this by hand or let electron-patcher do it? This second part shouldn't be here I think.

Reply:
Did this with:
npx patch-package app-builder-lib

Reply:
Maybe we can just safely remove that part?

Reply:
I think so, yeah. Can you please double-check though?

Reply:
Verified with https://travis-ci.com/github/freifunkMUC/jitsi-meet-electron/jobs/342565173#L450

And committed